sdk/azidentity/interactive_browser_server.go

```diff
@@ -70,7 +72,10 @@ func (s *server) Start(reqState string, port int) string {
 			if s.err != nil {
 				page = failPage
 			}
-			w.Write([]byte(page)) // nolint:errcheck
+			_, err := w.Write([]byte(page))
+			if err != nil {
+				page = failPage //nolint:ineffassign
```

Comment:
What is this doing? Seems like it was copied from above, but the above error handler is to set the failPage for writing. In case the actual write fails I'd assume you want to handle that too somehow (or ignore as before), as opposed to re-setting the variable which is then never used?

Reply:
Agree with Ben about this actually. This assignment is ineffectual here.